code/_helpers/global_lists_vr.dm

```diff
@@ -850,3 +850,98 @@ var/global/list/xenobio_rainbow_extracts = list(
 										/obj/item/slime_extract/emerald = 3,
 										/obj/item/slime_extract/light_pink = 1,
 										/obj/item/slime_extract/rainbow = 1)
+<<<<<<< HEAD
+=======
+
+
+// AREA GENERATION AND BLUEPRINT STUFF BELOW HERE
+// typecacheof(list) and list() are two completely separate things, don't break!
+
+// WHATEVER YOU DO, DO NOT LEAVE THE LAST THING IN THE LIST BELOW HAVE A COMMA OR EVERYTHING EVER WILL BREAK
+// ENSURE THE LAST AREA OR TURF LISTED IS SIMPLY "/area/clownhideout" AND NOT "/area/clownhideout," OR YOU WILL IMMEDIATELY DIE
+
+// These lists are, obviously, unfinished.
+
+// ALLOWING BUILDING IN AN AREA:
+// If you want someone to be able to build a new area in a place, add the area to the 'BUILDABLE_AREA_TYPES' and 'blacklisted_areas'
+// BUILDABLE_AREA_TYPES means they can build an area there. The blacklisted_areas means they CAN NOT EXPAND that area. No making space bigger!
+
+// DISALLOW BUILDING/AREA MANIPULATION IN AN AREA (OR A TURF TYPE):
+// Likewise, if you want someone to never ever EVER be able to do anything area generation/expansion related to an area
+// Then add it to SPECIALS and area_or_turf_fail_types
+
+// If you want someone to
+var/global/list/BUILDABLE_AREA_TYPES = list(
+	/area/space,
+	/area/mine,
+//	/area/surface/outside, 	//SC
+//	/area/surface/cave,		//SC
+	/area/tether/surfacebase/outside,
+	/area/groundbase/unexplored/outdoors,
+	/area/maintenance/groundbase/level1,
+	/area/submap/groundbase/wilderness,
+	/area/groundbase/mining,
+	/area/offmap/aerostat/surface,
+	/area/tether_away/beach,
+	/area/tether_away/cave,
+)
+
+var/static/list/blacklisted_areas = typecacheof(list(
+	/area/space,
+	/area/mine,
+//	/area/surface/outside,	//SC
+//	/area/surface/cave,		//SC
```

Comment:
Uncomment this.

Reply:
Actually I don't think these are ours either
![image](https://user-images.githubusercontent.com/15962836/193506867-a15a8464-dc1f-4376-99a5-5dd38543d19a.png)